.github/scripts/trymerge.py

```diff
@@ -990,7 +990,7 @@ def fetch_check_run_conclusions(repo: GitRepo, commit: str) -> Dict[str, Tuple[s
     [owner, name] = repo.gh_owner_and_name()
     checks = fetch_json_dict(f'https://api.github.com/repos/{owner}/{name}/commits/{commit}/check-runs')
```

Comment:
I couldn't find a way to get from commit -> check_suites in the graphql API https://docs.github.com/en/graphql/reference/objects#repository. 

All the edges to commits are from things that I can't really use. We can't use the pull request edge cause the checks are going to be on a different branch that's not gonna be merged in. LMK if you see an easier way to do it though; might be missing something.

Reply:
So in discussion we mentioned that REST API has low api limits. I'll continue with this since the API rate limit shouldn't be hit too often but we'll look into switching it into the GraphQL API

Reply:
https://github.com/pytorch/pytorch/blob/1d4dbdcefe55c8cbee1aaca960f9b5568dd66b6c/.github/scripts/trymerge.py#L184